quam_builder/architecture/quantum_dots/virtual_gates/README.md

```diff
@@ -0,0 +1,231 @@
+# VirtualGateSet: Abstracting Gate Control with Virtualisation Layers
+
+## 1. Introduction
+
+This document introduces the **VirtualGateSet** and **VirtualisationLayer** components. These extend the physical gate control capabilities provided by `GateSet` and `VoltageSequence` by adding one or more layers of virtual gates.
+
+Virtual gates simplify complex tuning procedures in experiments, especially for spin qubits, by allowing control over abstract parameters that map to multiple physical gate voltages.
+
+**`VirtualGateSet` builds upon `GateSet` and inherits all of its features**, including physical channel management, `VoltageTuningPoint` definitions, and all voltage control methods. This means you can use virtual gates while still having access to all the voltage control capabilities of the underlying `GateSet`.
+
+## 2. Core Components
+
+### 2.1 VirtualGateSet
+
+A subclass of `GateSet`. It manages a list of `VirtualisationLayer` objects, which define the transformations from virtual gate voltages to underlying (either physical or lower-level virtual) gate voltages.
+
+### 2.2 VirtualisationLayer
+
+Represents a single linear transformation (matrix) from a set of source (virtual) gates to a set of target gates.
```

Comment:
I think that adding the mathematical relation between the target virtual gets and the source one would be very useful.

Reply:
added